indy-utils/src/main/kotlin/com/luxoft/blockchainlab/hyperledger/indy/utils/EnvironmentUtils.kt

```diff
@@ -17,24 +12,31 @@ internal object EnvironmentUtils {
             return testPoolIp ?: "127.0.0.1"
         }
 
+    //Should be similar to  RUST`s implementation
+    private val userHomePath: String get() = System.getenv("HOME")
+
     fun getIndyHomePath(): String {
-        return getUserDirectoryPath() + "/.indy_client/"
+        return System.getProperty("INDY_HOME") ?: "$userHomePath/.indy_client"
+    }
+
+    fun getIndyPoolPath(): String {
+        return System.getProperty("INDY_POOL_PATH") ?: "$userHomePath/.indy_client"
     }
 
-    fun getIndyPoolPath(poolName: String) = getIndyHomePath() + "/pool/$poolName"
+    fun getIndyPoolPath(poolName: String) = getIndyPoolPath() + "/pool/$poolName"
```

Comment:
This method (and some others) store a system path as a string in Unix format. This code probably won't on Windows. Is there a good reason not to support Windows (such as some core underlying technology not supporting Windows)? 

A possibly fix could be replacing all string-based paths with JDK `Path` and using its operations such as path concatenation or navigation.

Reply:
In general case you ofc right, but.
As you can see in Hyperledger Indy's Jira there are many bugs and unexpected behavior on Windows right now.
https://jira.hyperledger.org/browse/IS-259?jql=project%20%3D%20%22IS%22%20AND%20text%20~%20windows

My opinion is - right now we should give up bothering Windows compatibility at all. When the day comes we'll start some activity to make whole Cordentity run on Windows.
Why do I think so? Even if Alexey fix these paths right now it doesn't mean that everything else is Windows-compatible too.

Reply:
So technically, from the specification point of view, Indy is supported on Windows, and by extension, Cordentity should support Windows as well. Following that logic, this issue in the code is a bug, even if it hasn't been reported yet.

Frankly, I wouldn't merge a problematic code without a higher business need. On the other hand you of course is right that this code won't become problematic for some time, so we can file an issue and forget about it. The question is if having unreliable code is worth it when the fix is so simple :thinking:

Reply:
This code was already in develop, I`was building on top of it.
The fact that indy support windows doesn't mean that cordentity does.

Reply:
> The fact that indy support windows doesn't mean that cordentity does.

Do we specify it in the project description?

Reply:
We don`t have any information about supported platforms)

Reply:
In general. I would suggest to move to File() object related path handling, but that`s definitely not what current PR covers. Because there is not point in doing it without testing on multiple platforms including windows.

Reply:
I'm agree with Daniil, there is no needs to explicitly include known bugs for Win. We have platforms limitation anyway, but at the same time we can reduce number of issues to be compatible in future. 

It's fine to make it in separated PR